src/processors/operation-processor.ts

```diff
@@ -122,6 +150,14 @@ export default class OperationProcessor<ResourceT extends Resource> {
         ...computedAttributes
       }
     });
+
+    const passesFilters = await this.matchesComputedFilters(op, computedAttributes);
+
+    if (!passesFilters) {
+      resource.preventSerialization = true;
+    }
+
+    return resource;
```

Comment:
Found a bug, I think related to this:
In the test app, if you filter by computed resources and include relations:
`localhost:3000/articles?filter[voteCount]=lt:1&include=author`
It brings all the included relations, including the ones of resources that don't pass the filter.
```
{
    "data": [
        {
            "id": 2,
            "type": "article",
            "attributes": {
                "body": "this is test 2",
                "voteCount": 0
            },
            "relationships": {
                "author": {
                    "data": {
                        "id": 2,
                        "type": "user"
                    }
                }
            }
        }
    ],
    "included": [
        {
            "id": 1,
            "type": "user",
            "attributes": {
                "username": "me",
                "email": "me@me.com",
                "createdAt": null,
                "updatedAt": null
            },
            "relationships": {}
        },
        {
            "id": 2,
            "type": "user",
            "attributes": {
                "username": "username2",
                "email": "me2@me.com",
                "createdAt": null,
                "updatedAt": null
            },
            "relationships": {}
        },
        {
            "id": 1,
            "type": "user",
            "attributes": {
                "username": "me",
                "email": "me@me.com",
                "createdAt": null,
                "updatedAt": null
            },
            "relationships": {}
        },
        {
            "id": 2,
            "type": "user",
            "attributes": {
                "username": "username2",
                "email": "me2@me.com",
                "createdAt": null,
                "updatedAt": null
            },
            "relationships": {}
        },
        {
            "id": 1,
            "type": "user",
            "attributes": {
                "username": "me",
                "email": "me@me.com",
                "createdAt": null,
                "updatedAt": null
            },
            "relationships": {}
        },
        {
            "id": 2,
            "type": "user",
            "attributes": {
                "username": "username2",
                "email": "me2@me.com",
                "createdAt": null,
                "updatedAt": null
            },
            "relationships": {}
        }
    ]
}
```

Reply:
This happens, I believe, because the base resource serialization step it's done _after_ the included resources serialization.

Reply:
I'll take a look at it.

Reply:
I'm not able to reproduce this with the dummy app. I used the exact same request and got this:

![image](https://user-images.githubusercontent.com/118913/97937592-e1a84f00-1d5d-11eb-8157-1743ff68b77e.png)